packages/slice-machine/src/features/customTypes/actions/renameCustomType.ts

```diff
@@ -0,0 +1,46 @@
+import { CustomType } from "@prismicio/types-internal/lib/customtypes";
+import { managerClient } from "@src/managerClient";
+import { CUSTOM_TYPES_MESSAGES } from "../customTypesMessages";
+import { CustomTypeFormat } from "@slicemachine/manager/*";
```

Comment:
```suggestion
import { CustomTypeFormat } from "@slicemachine/manager";
```

When important manager stuff it's by default like that, bad conf. Not necessary and strange.